src/containers/career.jsx

```diff
@@ -24,14 +24,49 @@ class Career extends Component {
 
     this.state = {
       currentIndex: 0,
-      careers: []
+      careers: [
+        {
+          title: ' ',
+          image: ' ',
+          description: ' ',
+          degree: ' ',
+          grade_bagrut: ' ',
+          grade_psychometric: ' ',
+          universities: ' ',
+          salary_start: ' ',
+          salary_ten_year: ' '
+        }
+      ]
     };
     this.addToLocal = this.addToLocal.bind(this);
     this.nextCareer = this.nextCareer.bind(this);
     this.flipCard = this.flipCard.bind(this);
     this.flipCardBack = this.flipCardBack.bind(this);
   }
 
+  componentWillReceiveProps(nextProps) {
+    if (nextProps.careers !== this.props.careers) {
+      if (localStorage.liked) {
+        const likedCareers = JSON.parse(localStorage.liked);
+        const allCareers = nextProps.careers;
+        const notLikedCareers = _.differenceBy(allCareers, likedCareers, 'title');
```

Comment:
i'm not big on lodash can you explain what this does?

Reply:
google it m8 :wink:

Reply:
[differenceBy](https://lodash.com/docs/4.17.4#differenceBy)